src/main/java/ca/corefacility/bioinformatics/irida/ria/web/components/agGrid/AgGridColumn.java

```diff
@@ -166,11 +166,11 @@ public void setHeaderCheckboxSelection(boolean headerCheckboxSelection) {
 		this.headerCheckboxSelection = headerCheckboxSelection;
 	}
 
-	public boolean isFilter() {
+	public String isFilter() {
```

Comment:
Can you rename this to `getFilter`?  `isFilter` semantics say that's a boolean.

Reply:
Fixed in e7241ee69920b5a64b5dcc93be17ee771cb4e513